submissions/yuryzaytsev/css/style.css

```diff
@@ -0,0 +1,685 @@
+    width: 40px;
+    height: 40px;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: center;
+        -ms-flex-pack: center;
+            justify-content: center;
+    -webkit-box-align: center;
+        -ms-flex-align: center;
+            align-items: center;
+    overflow: hidden; }
+
+.header__link {
+  font-weight: 600;
+  font-size: 14px;
+  line-height: 17px;
+  text-transform: uppercase;
+  text-decoration: none;
+  letter-spacing: 0.03em;
+  outline: none;
+  -webkit-transition: color 0.1s linear;
+  transition: color 0.1s linear; }
+  .header__link svg {
+    fill: #000000; }
+  .header__link:hover, .header__link:focus {
+    color: #e75333; }
+  .header__link--user img {
+    border-radius: 50%; }
+
+.header__btn {
+  position: relative;
+  display: none;
+  height: -webkit-max-content;
+  height: -moz-max-content;
+  height: max-content;
+  margin: 0;
+  padding: 0;
+  background-color: transparent;
+  border: none;
+  outline: none; }
+
+.footer {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+      -ms-flex-pack: center;
+          justify-content: center;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  background-color: #000000; }
+
+.footer__item {
+  margin: 7px; }
+
+.footer__link {
+  font-size: 12px;
+  font-weight: 500;
+  -webkit-transition: color 0.1s linear;
+  transition: color 0.1s linear;
+  text-decoration: none;
+  color: #ffffff;
+  outline: none; }
+  .footer__link:hover, .footer__link:focus {
+    color: #e75333; }
+
+.page {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto 1fr auto;
+  min-height: 100%;
+  min-width: 320px; }
+
+.popup {
+  position: absolute;
+  display: none;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+      -ms-flex-direction: column;
+          flex-direction: column;
+  top: 65px;
+  border-radius: 5px;
+  background-color: #ffffff;
+  -webkit-box-shadow: 2px 2px 1px rgba(41, 52, 73, 0.5);
+          box-shadow: 2px 2px 1px rgba(41, 52, 73, 0.5);
+  max-height: 400px;
+  overflow-y: auto;
+  -webkit-transform: translateX(-50%);
+          transform: translateX(-50%);
+  left: 50%; }
+  .popup::-webkit-scrollbar {
+    width: 3px; }
+  .popup::-webkit-scrollbar-thumb {
+    background-color: rgba(0, 0, 0, 0.3);
+    border-radius: 0px; }
+  .popup::-webkit-scrollbar-thumb:hover {
+    background-color: rgba(0, 0, 0, 0.5); }
+  @media (min-width: 768px) {
+    .popup {
+      -webkit-transform: initial;
+              transform: initial;
+      left: initial;
+      right: 10px; } }
+
+.popup #btn-more:focus ~ .popup__btn {
+  background-color: rgba(0, 0, 0, 0.3); }
+
+.popup #btn-more:checked ~ .popup__btn {
+  display: none; }
+
+.popup #btn-more:checked ~ .popup__list--hidden {
+  display: grid; }
+
+.popup__list {
+  display: grid;
+  gap: 5px;
+  grid-template-columns: 90px 90px 90px;
```

Comment:
In this case preferable to use `repeat()` function. 
[Read more about it](https://developer.mozilla.org/en-US/docs/Web/CSS/repeat).

Reply:
Fixed.